### DISCUSSION TITLE
im approved as a student for student developer pack and yet its not reflecting in my account . HELP!!!!

### DISCUSSION THREAD

**Discussion by Shanker-0412** [community/community#112721] [GitHub Education] [answered]
### Hi everyone,

ive applied for student developer pack and i got approved on dat day itself but for some reason its not reflecting in my account. it still shows github free account even though i got approved from github
![Screenshot 2024-03-16 161142](https://github.com/community/community/assets/163419612/4ec45768-2d9b-4457-9f5d-b292f8c64f93)

**Comment by ghost** [comment]
Hi @Shanker-0412 it takes time to fully get the benefits. usually few days after approval.
as it is saying "your benefits will be available soon".
GL.

**Comment by SebastienXia** [comment, marked as answer]
## Hellow, @Shanker-0412 👋

🙌 Don't worry a lot of student/prof are in the same situation, you just need to wait 6 - 7 days after been approved in the process !

`[close the thread and mark as answered if this helped you resolve your issue]`

**Comment by SebastienXia** [comment]
> You will also receive an email when you have access to the benefits of the Pack

**Reply by Shanker-0412** [reply]
thanks. hope i get it asap

**Reply by kfofkf123** [reply]
yeah, me as well
